### PR TITLE
Add linter action

### PR DESCRIPTION
**Pull request by NirmalAgnihotri**
### Tracking Info

Resolves #<issue number>

Make sure your branch name conforms to: `<feature|staging|bugfix|...>/<username>/<3-4 word description separated by dashes>`. Otherwise, please rename your branch and create a new PR.

### Changes

What changes did you make?

- Added linting on push action
### Testing

How did you confirm your changes worked? 

- TODO

### Confirmation of Change 

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

TODO...

### CODE REVIEW

**Review by vs2961 — CHANGES_REQUESTED** [review]
Your github action seems to be failing, perhaps because this was done on a branch where eslint wasn't installed? In any case, I'd recommend merging this PR with #24 because they are pretty similar changes.

File: server/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "commonjs": true,
+    "es2021": true
+  },
+  "extends": "standard",
+  "overrides": [],
+  "parserOptions": {
+    "ecmaVersion": "latest"
+  },
+  "rules": {
+    // Disable rules that conflict with Mongo
```
Comment (vs2961) [inline server/.eslintrc.json:13]: These rules work for me!

File: SDCTA/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "es2021": true
+  },
+  "extends": ["plugin:react/recommended", "standard-with-typescript"],
+  "overrides": [],
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["react"],
```
Comment (vs2961) [inline SDCTA/.eslintrc.json:13]: Honestly don't know what the correct rules to ignore are for the frontend, so I'll leave that to someone working on a frontend team.

**Review by vs2961 — APPROVED** [review]
lgtm, if something changed in the other files I'll probably never catch it anyways so very good!

File: SDCTA/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "@typescript-eslint/quotes": 0,
+    "@typescript-eslint/semi": 0,
+    "@typescript-eslint/comma-dangle": 0,
+    "@typescript-eslint/member-delimiter-style": [
+      "error",
+      {
+        "multiline": {
+          "delimiter": "semi", // 'none' or 'semi' or 'comma'
+          "requireLast": true
+        }
```
Comment (Anthonyp0329) [inline SDCTA/.eslintrc.json:26]: Rules look good to me! Can also change as we go if necessary

**Review by Anthonyp0329 — APPROVED** [review]

**Pull request merged**